lib/profiles/dosomething/drupal-org.make

```diff
@@ -253,6 +253,10 @@ projects[stage_file_proxy][subdir] = "contrib"
 ;libraries[dsapi][download][type] = "git"
 ;libraries[dsapi][download][url] = "git@github.com:DoSomething/dsapi-php.git"
 
+; Guzzle
+libraries[guzzle][download][type] = "git"
+libraries[guzzle][download][url] = "https://github.com/guzzle/guzzle.git"
```

Comment:
+1 to locking to a specific tag. Seems inevitable that we'll get burned when they release a major version otherwise.